### PR TITLE
Update "Setting up a project" docs

### PR DESCRIPTION
**Pull request by minwoox**
The page is outdated so we need to update it.

### CODE REVIEW

File: site/src/pages/docs/setup.mdx [hunk 1]
```diff
@@ -116,7 +141,6 @@ You might want to use the following build file as a starting point when you set
         }\n
         // Logging
         runtimeOnly 'ch.qos.logback:logback-classic:${versions['ch.qos.logback:logback-classic']}'
-        runtimeOnly 'org.slf4j:log4j-over-slf4j:${versions['org.slf4j:log4j-over-slf4j']}'
```
Comment (minwoox) [inline site/src/pages/docs/setup.mdx:119]: Users do not have to specify slf4j because Armeria core uses it with `api` configuration.

**Review by jrhee17 — APPROVED** [review]
Thanks for the cleanup @minwoox ! 🙇 👍 🙇

File: site/src/pages/docs/setup.mdx (hunk 1 above)
Comment (ikhoon) [reply site/src/pages/docs/setup.mdx:119]: Would you also update L131, L133, L137, and L138?

Comment (minwoox) [reply site/src/pages/docs/setup.mdx:119]: Oops, fixed. Thanks!

**Review by ikhoon — APPROVED** [review]
Thanks, @minwoox!

**Pull request merged**